lib/client-assets.php

```diff
@@ -765,9 +770,14 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 		'colors'     => $color_palette,
 	);
 
-	wp_add_inline_script( 'wp-editor', 'wp.api.init().done( function() {'
-		. 'window._wpGutenbergEditor = wp.editor.createEditorInstance( \'editor\', window._wpGutenbergPost, ' . json_encode( $editor_settings ) . ' ); '
-		. '} );'
+	wp_add_inline_script(
+		'wp-editor',
+		(
+			'jQuery.when( wp.api.init(), wp.api.init( { versionString: \'gutenberg/v1/\' } ) ).done( function() {'
```

Comment:
Is `jQuery.when` the same as `Promise.all`. It would be great if we can avoid `jQuery`

Reply:
I expect `init` may be returning an instance of [`jQuery.Deferred`](https://api.jquery.com/jquery.deferred/). Unsure if `Promise.all` is capable of handling these.

Reply:
It looks like that would work: https://codepen.io/noisysocks/pen/zPROqr?editors=0011

Reply:
👌 fixed in b2e1a7d054c06e26a2586a846dc4124292a27fc7.

Reply:
Wait, we can't do this though because IE 11 doesn't support promises! https://caniuse.com/#feat=promises

Reply:
We use a promise polyfill :) https://github.com/WordPress/gutenberg/blob/6eec1886b9270073ab7cf6ca2543ad39640dafb1/lib/client-assets.php#L141

Reply:
Right you are! 😄 Changed in f5cd6cded675b9dec31d52ec961064f7a060fa04.